tests_e2e/tests_e2e.py

```diff
@@ -0,0 +1,61 @@
+import os
+from threading import Thread
+import pytest
+from selenium import webdriver
+from dotenv import load_dotenv
+import requests
+
+from todo_app.flask_config import Config
+from todo_app.app import create_app
+
+@pytest.fixture(scope="module")
+def driver():
+    with webdriver.Firefox() as driver:
+        yield driver
+
+def create_trello_board():
+    config = Config()
+    response = requests.post(
+        url=f'{config.TRELLO_BASE_URL}/boards',
+        params={
+            'key': config.TRELLO_KEY,
+            'token': config.TOKEN,
+            'name': 'Selenium Test Board'
+        }
+    )
+    return response.json()['id']
+
+
+def delete_trello_board(board_id):
+    config = Config()
+    requests.delete(
+        url=f'{config.TRELLO_BASE_URL}/boards/{board_id}',
+        params={
+            'key': config.TRELLO_KEY,
+            'token': config.TOKEN,
+        }
+    )
+
+@pytest.fixture(scope='module')
+def app_with_temp_board():
+    # Create the new board & update the board id environment variable
+    board_id = create_trello_board()
```

Comment:
You should explicitly load your env variables here before trying to create the board.
```python
load_dotenv(find_dotenv('.env'), override=True)
```
It happens to work at the moment as long as the integration tests don't run first because you call `load_dotenv()` in `session_items.py`